### DISCUSSION TITLE
Adding markdown to the content field

### DISCUSSION THREAD

**Discussion by abhiabhi94** [Radi85/Comment#118] [Ideas]
There are a couple of questions that probably need answers before any code is written:

- Is this really required? Will the feature be worth it?

- Should we use a third-party app for this? What options do we have? Pros and cons of each option.

- Should we avoid using a third party app? If yes, will implementing this by ourselves be worth it?

- Implementing this would require a bit of change in the way content is rendered as of now. (Content is trimmed after a certain length)

**Comment by Radi85** [comment]
> * Is this really required? Will the feature be worth it?

Why not adding such a feature and make it optional.  

> * Should we use a third-party app for this? What options do we have? Pros and cons of each option.
> * Should we avoid using a third party app? If yes, will implementing this by ourselves be worth it?

Why re-inventing the wheel? I would go for third-party and keep our concentration on the comment app only.
This [package](https://github.com/trentm/python-markdown2) seems to be eligible.

> * Implementing this would require a bit of change in the way content is rendered as of now. (Content is trimmed after a certain length)

We should make this feature optional and disabled by default. Enabling this feature should disable truncated text.
